artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -1,322 +1,114 @@
+.Procedures
 
-.  Add the acquired authentication key to the environment variables for your Backstage backend server as follows:
-+
---
-`ARGOCD_AUTH_TOKEN="argocd.token=<auth-token>"`
---
-
-== Support for multiple ArgoCD instances in Backstage
-
-There are two options to support multiple ArgoCD instances in Backstage, including:
-
-* Adding proxy configuration for each ArgoCD instance
-* Using the ArgoCD backend plugin
-
-.Option 1: Adding proxy configuration for each ArgoCD instance
+.Procedure
 
-. To create multiple components that fetch data from different ArgoCD instances, add a proxy configuration for each instance as shown in the following example:
+. Add the following to your dynamic-plugins ConfigMap to enable the Argo CD plugin.
 +
---
 [source,yaml]
 ----
-proxy:
-  ...
-
-  '/argocd/api':
-    target: https://<someAddress>/api/v1/
-    changeOrigin: true
-    secure: false
-    headers:
-      Cookie:
-        $env: ARGOCD_AUTH_TOKEN
-
-  '/argocd/api2':
-    target: https://<otherAddress>/api/v1/
-    changeOrigin: true
-    secure: false
-    headers:
-      Cookie:
-        $env: ARGOCD_AUTH_TOKEN2
-
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
+        disabled: false
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-argocd
+        disabled: false
 ----
---
 
-. Add the required authentication tokens to environment variables, such as `ARGOCD_AUTH_TOKEN2`.
-+
---
-After adding the authentication tokens, add the URL to the desired proxy path in the following component definition annotations:
+== For Users
 
-`argocd/proxy-url: '/argocd/api2'`
+.Prerequisites
 
-The `argocd/proxy-url` annotation defaults to `'/argocd/api'`. Therefore, `argocd/proxy-url` is not required if there is a single proxy configuration.
---
+* You have enabled the Argo CD plugin in {product} {product-very-short}.
 
-.Option 2: Using the ArgoCD backend plugin
+.Procedures
 
-. To create multiple components that fetch data from different ArgoCD instances, set the ArgoCD instance URL dynamically by adding the following configuration to `app-config.yaml` file:
-+
---
-[source,yaml]
-----
-argocd:
-  username: ${ARGOCD_USERNAME}
-  password: ${ARGOCD_PASSWORD}
-  waitCycles: 25
-  appLocatorMethods:
-    - type: 'config'
-      instances:
-        - name: argoInstance1
-          url: https://argoInstance1.com
-          token: ${ARGOCD_AUTH_TOKEN} # optional
-        - name: argoInstance2
-          url: https://argoInstance2.com
-          # dedicated username/password for this instance
-          username: ${ARGOCD_USERNAME_INSTANCE_2} # optional
-          password: ${ARGOCD_PASSWORD_INSTANCE_2} # optional
+. On the {product-very-short} platform, navigate to the *Catalog* and select an appropriate component.
```

Comment:
Click the **Catalog** tab and select the component that you want to use.

Reply:
> Click the Catalog tab and select the component that you want to use.
Updated but raised a question below. Click vs Select. Kept select as for now